### PR TITLE
feat: add line-height ratio (#18)

### PR DESCRIPTION
**Pull request by csshugs**
because:
- usually, big font-sizes do have a smaller line-height (in
relation to the font-size) than “normal” font-sizes do

this commit:
- introduces a new Sass variable to control the ratio with which
the line-height gets decreased, the bigger the font-size is

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -193,6 +193,16 @@ You can define an ideal line-height that is taken into account when calculating
 $FLUIDMS-GLOBAL-LINE-HEIGHT: 1.5 !default;
 ```
 
+### Changing the global line-height ratio
+
+The bigger font-sizes get, the less big needs the line-height to be in relation to its font-size. Hence, we provide a an option to decrease the line-height in proportion to the font-size, the bigger the font-size gets.
```
Comment (fuhlig) [inline README.md]: ~~less big~~ the lesser the line-height needs to be

Comment (fuhlig) [inline README.md]: provide ~~a~~ an option

File: README.md [hunk 2]
```diff
@@ -193,6 +193,16 @@ You can define an ideal line-height that is taken into account when calculating
...
+
+```scss
+$FLUIDMS-GLOBAL-LINE-HEIGHT-RATIO: 1 !default;
+```
+
+Don’t get crazy with this variables value! Only nudge the ratio slightly and always test all the possible font-sizes with multiline text.
```
Comment (fuhlig) [inline README.md]: variable's

File: src/settings/_settings.config.scss [hunk 3]
```diff
@@ -25,6 +25,16 @@ $FLUIDMS-GLOBAL-BASELINE: 0.25rem !default;
 ///   $FLUIDMS-GLOBAL-LINE-HEIGHT: 1.35 !default
 $FLUIDMS-GLOBAL-LINE-HEIGHT: 1.5 !default;
 
+/// Define the ratio with which the line-height is decreased relatively for
+/// larger font-sizes. That is, the calculated line-height value is getting
+/// smaller in ralation to the font-size, the bigger the font-size gets.
+///
+/// @type number
+///
+/// @example scss - Sets the global line-height ratio to `1.1`
```
Comment (fuhlig) [inline src/settings/_settings.config.scss]: example serves no purpose IMO as the variable + value are self-explanatory

File: src/tools/_tools.line-height.scss [hunk 4]
```diff
@@ -21,7 +21,7 @@
 ///   our `$FLUIDMS-GLOBAL-BASELINE`.
 @function fluidms-line-height($scale, $ratio) {
     // Calculate the default line-height (in `rem`).
-    $_default-line-height: (fluidms-pow($ratio, $scale)) * $FLUIDMS-GLOBAL-LINE-HEIGHT;
+    $_default-line-height: (fluidms-pow($ratio, $scale)) * ($FLUIDMS-GLOBAL-LINE-HEIGHT / fluidms-pow($FLUIDMS-GLOBAL-LINE-HEIGHT-RATIO, $scale));
```
Comment (fuhlig) [inline src/tools/_tools.line-height.scss:24]: Not part of this PR but concerning the variables I'd recomment to remove the `GLOBAL` from the variable name as the uppercase symbolizes the global scope (document that)

**Review by fuhlig — CHANGES_REQUESTED** [review]

File: src/tools/_tools.line-height.scss (hunk 4 above)
Comment (csshugs) [reply src/tools/_tools.line-height.scss:24]: 1. I agree that this is not part of this PR.
2. I’m not sure if I agree with removing the `GLOBAL` part from the variable(s). Yes, we have a convention that says, uppercase vars are global, but IMO it is more clear to leave the global namespace in there, to make that fact more clear. If you want, we can discuss this separately in a dedicated issue.

**Review by fuhlig — APPROVED** [review]
